rust/src/treefile.rs

```diff
@@ -541,6 +541,10 @@ impl Treefile {
         self.parsed.repos.clone().unwrap_or_default()
     }
 
+    pub(crate) fn get_ref(&self) -> &str {
+        self.parsed.treeref.as_deref().unwrap_or_default()
```

Comment:
Minor: `unwrap_or("")` just like `get_releasever`?

Reply:
The other surrounding code uses `unwrap_or_default()` though; I think the releasever one is the odd one out.